src/components/shared/Error/Message.tsx

```diff
@@ -0,0 +1,49 @@
+import { Box, Stack, Typography } from '@mui/material';
+import MessageWithLink from 'components/content/MessageWithLink';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { logRocketConsole } from 'services/logrocket';
+import { ErrorDetails } from './types';
+
+interface Props {
+    error?: ErrorDetails;
+}
+
+function Message({ error }: Props) {
+    logRocketConsole('Error message displayed', error);
```

Comment:
Threw this in so while we debug in prod we could still get access to all the error details (in case there are any details missing)